Avoid passing -L/lib -L/usr/lib while linking binaries, it is unnecessary and wrong
--- libreoffice-6.4.2.2/solenv/gbuild/platform/solaris.mk.1	2020-04-09 01:51:33.146199992 +0000
+++ libreoffice-6.4.2.2/solenv/gbuild/platform/solaris.mk	2020-04-09 01:51:43.369737802 +0000
@@ -68,8 +68,6 @@
 endif
 
 gb_LinkTarget_LDFLAGS += \
-	-L$(SYSBASE)/lib \
-	-L$(SYSBASE)/usr/lib \
 	-Wl,-B,direct \
 	-Wl,-z,defs \
 	-Wl,-z,combreloc \
